apps/website/src/data/scheduled-tasks.ts

```diff
@@ -72,6 +64,6 @@ const config: ScheduledTasksConfig = {
   ],
 };
 
-export async function getScheduledTasksConfig() {
+export function getScheduledTasksConfig() {
```

Comment:
Does this even need to be a method at this point, we could just export default the config, so we're not creating new object instances all the time?

Reply:
I thought that as well. I didn't want to make such a change in case it was some convention this repo is following. We could give a more descriptive name to the config and export it or just export default as you suggested.

Reply:
> Does this even need to be a method at this point, we could just export default the config, so we're not creating new object instances all the time?

Does it create new object instances with each invocation? I think it always references the same instance, but the change would still go along with the spirit of "Reduce code. Improve maintainability." from the issue.

Reply:
yeah go for it. its not creating new instances or anything.

Reply:
Any preference for how the config is exported?

Reply:
I think we have a mix of default export and named exports. I do not really mind either way. I'd probably name it `scheduledTasks` i guess 🤷  We do not really need the `tasks` object either. Can just be the array of tasks.

Reply:
> Does it create new object instances with each invocation? I think it always references the same instance

I think you're right, but it probably creates a new closure every time still.